src/entries/popup/pages/home/TokenDetails/TokenDetails.tsx

```diff
@@ -284,39 +294,58 @@ function MoreOptions({ token }: { token: ParsedUserAsset }) {
         </div>
       </DropdownMenuTrigger>
       <DropdownMenuContent align="end">
-        <AccentColorProvider
-          color={token.colors?.primary || token.colors?.fallback}
-        >
-          {!isNative && (
-            <DropdownMenuItem
-              symbolLeft="doc.on.doc.fill"
-              onSelect={() => copyAddress(token.address)}
+        {swappable && (
+          <Fragment>
```

Comment:
if AccentColorProvider wraps it all do we need the fragment?

Reply:
Oohh great catch 👀. Fixed it in this [commit](https://github.com/rainbow-me/browser-extension/pull/1402/commits/85a314b3b5f1cb1c8a7c1be6cabe738f41ac7909).